yt/frontends/amrvac/fields.py

```diff
@@ -162,6 +161,7 @@ def dust_to_gas_ratio(field, data):
             )
 
     def setup_fluid_fields(self):
+        from yt.fields.magnetic_field import setup_magnetic_field_aliases
```

Comment:
what's the reason for moving the import statements (here and in the other frontend fields.py files)? Doesn't seem related to the focus of the PR. That said, it's such a small change that I don't mind tacking it on here, I'd just like to understand the change a bit more...

Reply:
I took the opportunity to go thought [the whole page about frontend development](https://yt-project.org/docs/dev/developing/creating_frontend.html) and found [this paragraph](https://yt-project.org/docs/dev/developing/creating_frontend.html#creating-aliases-for-magnetic-fields), and most importantly this sentence:
> This function should always be imported and called from within the setup_fluid_fields method of the FieldInfoContainer.

So TBH I don't know _why_ this is the recommendation, but I noticed a misalignment with the reality of existing frontends and picked the easiest way to make them aligned again.

Reply:
Oh interesting. I wonder why. in any case, makes sense now, thanks!